config/install/oe_authentication.settings.yml

```diff
@@ -3,3 +3,4 @@ register_path: 'eim/external/register.cgi'
 validation_path: 'TicketValidationService'
 assurance_level: TOP
 ticket_types: SERVICE,PROXY
+force_2fa: false
```

Comment:
I assume we need to provide update path to set up this option for existing sites.